src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,131 @@
 package core.basesyntax;
 
+import java.util.Map;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int size;
+    private int capacity;
+    private int threshold;
+
+    @SuppressWarnings("unchecked")
+    public MyHashMap() {
+        table = (Node<K, V>[]) new Node[DEFAULT_CAPACITY];
+        size = 0;
+        capacity = DEFAULT_CAPACITY;
+        threshold = (int) (DEFAULT_CAPACITY * DEFAULT_LOAD_FACTOR);
+    }
 
     @Override
     public void put(K key, V value) {
-
+        resize();
```

Comment:
Maybe it's will be beter if you add here if-statement, not in the method resize

Reply:
i'm told about this :
```suggestion
if (size >= threshold) {
    resize();
}
```